### DISCUSSION TITLE
Browser tab name

### DISCUSSION THREAD

**Discussion by Dailoi2202** [refinedev/refine#5442] [Q&A] [answered]
Hi, does anyone know how to set browser tab name

**Comment by aliemir** [comment, marked as answer]
Hey @Dailoi2202 I think you mean window title;

Check out [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/API/Document/title) to learn more.

Refine offers automated way of managing document titles depending on resource and actions of the current route via router integrations.

- [`DocumentTitleHandler` in `@refinedev/react-router-v6`](https://refine.dev/docs/routing/integrations/react-router/#documenttitlehandler)
- [`DocumentTitleHandler` in `@refinedev/nextjs-router`](https://refine.dev/docs/routing/integrations/next-js/#documenttitlehandler)
- `@refinedev/remix-router` doesn't offer such component due to Remix handling title a bit differently, check out their docs for more info.
